gameView.html

```diff
@@ -55,6 +55,20 @@
         function west() {
             controller.moveWest();
         }
+
+        // Install keyboard handler for arrow keys
+        $(window).keypress(function(event) {
+            if (event.key === "ArrowUp" || event.key === 'w' || event.key === 'W') {

```

Comment:
Arrow keys aren't working on Linux/Chrome. Other keys work, though sucks for Dvorak :-P, but that's my own issue. Not marking needs fixed yet, but I wonder what is wrong with the arrow keys.

Reply:
I was jokingly thinking of adding "HJKL" support for all of you vim fans, too.

It should be easy enough to improve compatibility with Linux/Chrome. I only tested on Mac/Firefox. If I had this keypress handler in a debugger, I could inspect the event object. That event.key member worked perfectly for my configuration, but there's a lot of data in that event, including "event.which" which contained alternative ways of identifying keys. event.which had some ASCII numbers and some non-ASCII numbers for things like F2 keys and maybe arrows.